### PR TITLE
Fix missing audio artwork metadata on multiple exports

### PR DESCRIPTION
**Pull request by sebdesign**
| Q                  | A
| ------------------ | ---
| Bug fix?           | yes
| New feature?       | no
| BC breaks?         | no
| Deprecations?      | no
| Fixed tickets      | -
| Related issues/PRs | #281 
| License            | MIT

#### What's in this PR?

This commit fixes this issue by copying (by value) the `metaArr`
property to a local variable each time the filter is applied.

#### Why?

When the `AddMetadataFilter` is applied on an `Audio` media, the `artwork`
parameter was being unset from the `metaArr` array.

Saving the same `Audio` instance multiple times generated the
correct commands for the first export, but the subsequent commands were
missing the `artwork` parameter.

#### Example Usage

```php
$ffmpeg = new FFMpeg\FFMpeg::create();

$audio = $ffmpeg->open('track.wav');

$audio->addMetadata([
    "title" => "Some Title",
    "album" => "Some Album",
    "artwork" => "/path/to/some/file.jpg"
]);

// Now we can do

$mp3 = new FFMpeg\Format\Audio\Mp3();

// The first time we export a 320kbps MP3

$mp3->setAudioKiloBitrate(320);
$audio->save($mp3, 'track-hd.mp3');

// The second time we export a 128kbps MP3

$mp3->setAudioKiloBitrate(128);
$audio->save($mp3, 'track-sd.mp3');

### CODE REVIEW

**Comment by Romain** [comment]
Nice one. I'll merge it.

File: tests/Unit/Filters/Audio/AudioMetadataTest.php [hunk 1]
```diff
@@ -41,6 +41,7 @@ public function testAddArtwork()
         $filters = new AudioFilters($audio);
         $filters->addMetadata(array('genre' => 'Some Genre', 'artwork' => "/path/to/file.jpg"));
         $this->assertEquals(array(0 => "-i", 1 => "/path/to/file.jpg", 2 => "-map", 3 => "0", 4 => "-map", 5 => "1", 6 => "-metadata", 7 => "genre=Some Genre"), $capturedFilter->apply($audio, $format));
+        $this->assertEquals(array(0 => "-i", 1 => "/path/to/file.jpg", 2 => "-map", 3 => "0", 4 => "-map", 5 => "1", 6 => "-metadata", 7 => "genre=Some Genre"), $capturedFilter->apply($audio, $format));
```
Comment (jens1o) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: but why is this line here twice?

Comment (sebdesign) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: @jens1o The second line would make the test to fail with the previous implementation, because of the missing `artwork` parameter. Now with this PR, this assertion passes.

Comment (jens1o) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: @sebdesign I don't see any difference between 43 and 44? You'd not removed a line.

Comment (sebdesign) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: @jens1o I meant that the `artwork` parameter was missing from the array the `apply` method returns, which is a bug.

Comment (jens1o) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: Yeah, and I am talking about this line(which is useless). Anyway, it doesn't bother us in any way.

Comment (sebdesign) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: Not that's not useless, that's what I'm trying to explain. The *first* time the `apply` method was called the assertion passed. But if you wanted to `apply` again the same filter for a *second* time (e.g. to export to another format), then that line would make the test fail, because it would return a different array.

Comment (jens1o) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: I'm dumb. Thanks...

I'm so sorry. I'd forgot the context...

Comment (sebdesign) [inline tests/Unit/Filters/Audio/AudioMetadataTest.php]: No worries! Thank you for your feedback. :)
